kubernetes/coredns.yaml.sed

```diff
@@ -97,9 +99,20 @@ spec:
           operator: "Exists"
       nodeSelector:
         beta.kubernetes.io/os: linux
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Thanks @rdxmd, using `requiredDuringSchedulingIgnoredDuringExecution` makes sense.  I think we should also remove `replicas` from the Deployment.  It was originally set this way, but we later set replicas to 2 based on advice given in [this issue](https://github.com/coredns/deployment/issues/13).

FWIW, the kube-dns deployment does not set replicas to avoid conflict between autoscalers and add-on manager reconciliation.

Reply:
I see. And I also see, you had a long discussion about running on master or not in https://github.com/kubernetes/kops/pull/5917/files ...

Seems like there isn't a recommended setup for running coredns in kubernetes and one will have to look what is great for them. However, preparing a good base-setup like discussed above seems to be great to easy scale horizontal which whatever tools someone wants.